src/main.c

```diff
@@ -1630,12 +1641,18 @@ GOptionEntry entries_install[] = {
 };
 
 GOptionEntry entries_info[] = {
-	{"no-verify", '\0', 0, G_OPTION_ARG_NONE, &info_noverify, "disable bundle verification", NULL},
+	{"no-verify", '\0', 0, G_OPTION_ARG_NONE, &noverify, "disable bundle verification", NULL},
 	{"output-format", '\0', 0, G_OPTION_ARG_STRING, &output_format, "output format", "FORMAT"},
 	{"dump-cert", '\0', 0, G_OPTION_ARG_NONE, &info_dumpcert, "dump certificate", NULL},
 	{0}
 };
 
+GOptionEntry entries_resign[] = {
+	{"no-verify", '\0', 0, G_OPTION_ARG_NONE, &noverify, "disable bundle verification", NULL},
+	{"allow-expired", '\0', 0, G_OPTION_ARG_NONE, &resign_allow_expired, "use bundle signing time during verification", NULL},
```

Comment:
We need to decide what this option does, allow expired signatures or use the bundle singing time. This help message would be confusing